project/app/assets/stylesheets/boards/show.scss

```
@@ -1 +1,63 @@
 // ボード詳細画面（兼スレッド一覧画面）のスタイル
+body {
+    main.thread{
```

@y-kohei1228 
[should]
要素を柔軟に変更できるのよう、要素セレクタは取ってしまってください
ほかの箇所も同様です

```suggestion
    .thread{
```